### PR TITLE
[Justin] - Sage - How to use Witnet Oracle to display cryptocurrency prices

### PR DESCRIPTION
**Pull request by cjustinobi**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [x] The title of the Pull Request matches the title of my tutorial.
- [x] I've followed the tutorial template structure.
- [x] I've proofread the tutorial for spelling and grammar mistakes.
- [x] I've thoroughly explained the code, with inline code comments and text explanations.
- [x] I've run a linter & prettifier to format my code blocks.
- [x] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices.md [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+   uint256 public lastUpdateTime;
+
+   constructor(IWitnetPriceRouter _router, bytes4[] memory _ID4) {
+       witnetPriceRouter = _router;
+       ID4 = _ID4;
+       lastUpdateTime = block.timestamp;
+       updatePriceFeeds();
+   }
+
+   function updatePriceFeeds() public {
```
Comment (Julius170) [inline blog/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices.md:65]: can you check out your code base, its kinda buggy, I mean the "updatePriceFeeds" function specifically, theres an error while compiling, can you look into it

File: blog/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices.md [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+
+We built an NFT marketplace for tech artisans to connect with their prospective clients with a metric to highlight freelancers with frequent sales. [Here](https://witnet-oracle.vercel.app/) is the final demo of what we built. [Here](https://github.com/cjustinobi/witnet-oracle) is the link to the complete code.
+
+## About the Author​
+
+A software engineer, co-founder, Africinnovate, and a Web3 enthusiast. I used to call myself VueDetective. Connect with me on LinkedIn and Github
+
+## References​
+
+[https://docs.witnet.io/smart-contracts/witnet-data-feeds](https://docs.witnet.io/smart-contracts/witnet-data-feeds)
```
Comment (Julius170) [inline blog/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices.md:276]: Can you correct this link give it a better description than using the link itself as the description

**Review by Julius170 — COMMENTED** [review]
Heyyyy I left some comments, suggestive and corrective, you can check them out and make changes, also you forgot to add you header image to this PR, that's why it failed deployment 🥂

File: blog/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices.md [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+tags: [celosage, intermediate, react, hardhat, oracle]
+hide_table_of_contents: false
+slug: /tutorials/how-to-use-witnet-oracle-to-display-cryptocurrency-prices
+---
+
+![header](../../src/data-tutorials/showcase/intermediate/how-to-use-witnet-oracle-to-display-cryptocurrency-prices.png)
+
+## Introduction​
+
+There are a couple of oracle platforms one could use to get off-chain data to one's Dapp.
```
[the remark this answers is not in the document]
Comment (cjustinobi) [reply blog/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices.md:18]: Feedback well appreciated.

File: blog/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices.md (hunk 1 above)
Comment (cjustinobi) [reply blog/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices.md:65]: > can you check out your code base, its kinda buggy, I mean the "updatePriceFeeds" function specifically, theres an error while compiling, can you look into it

Recompiled from Remix and from the project, both are compiling. Did you create `.env` file and your REACT_APP_PRIVATE_KEY value?

File: blog/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices.md [hunk 4]
```diff
@@ -0,0 +1,277 @@
+---
+title: How to use Witnet oracle network to display cryptocurrency prices
+description: There are a couple of oracle platforms one could use to get off-chain data to one's Dapp. I want to show how to use Witnet to display price feeds of the popular blockchain networks.
```
[the remark this answers is not in the document]
Comment (cjustinobi) [reply blog/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices/2023-03-28-how-to-use-witnet-oracle-to-display-cryptocurrency-prices.md:3]: Done

**Pull request merged**